app/templates/partials/questions/errors.html

```diff
@@ -1,8 +1,9 @@
+{% import 'macros/helpers.html' as helpers %}
 <div class="panel panel--simple panel--error" data-qa="error">
   <div class="panel__header">
     {% if form.question_errors[question.id] is not none %}
     <ul class="list list--bare list--errors">
-      <li class="list__item venus" {{helpers.errorTracking(question.id, error)}}>{{ form.question_errors[question.id] }}</li>
+      <li class="list__item venus" {{helpers.track('error', 'Error', error, question.id)}}>{{ form.question_errors[question.id] }}</li>
```

Comment:
That would be more consistent but don't we want to have the error message (action) as well as the question.id (label)